doc/en/installation/common/install_packages.rst

```diff
@@ -86,11 +88,6 @@ Run the commands::
 .. note::
     **centreon-database** package installs a database server optimized for use with Centreon.
 
-.. note::
```

Comment:
we talked about strict mode in beta mode, didn't we ?

Reply:
in Release note